Aquality.WinAppDriver.Template/Aquality.WinAppDriver.Template/Forms/AboutTheAuthorWindow.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Aquality.WinAppDriver.Elements.Interfaces;
+using Aquality.WinAppDriver.Forms;
+using OpenQA.Selenium;
+
+namespace Aquality.WinAppDriver.Template.Forms
+{
+    public class AboutTheAuthorWindow : Window
+    {
+        private readonly IButton CloseButton;
+
+        public AboutTheAuthorWindow() : base(By.Name("About the Author"), "About the Author")
+        {
+            CloseButton = ElementFactory.FindChildElement<IButton>(this, By.Name("CLOSE"), "Close");
```

Comment:
we can just call FindChildElement method of the base class, without referencing of ElementFactory and without passing "this"